### ISSUE TITLE
Documentation and code do not agree on runDisposables

### ISSUE THREAD

**Issue by SanderKnauff** [ember-lifeline/ember-lifeline#977]
[According to the code](https://github.com/ember-lifeline/ember-lifeline/blob/master/addon/utils/disposable.ts#L54), runDisposables is deprecated and is not required anymore. This clashes with the [documentation](https://ember-lifeline.github.io/ember-lifeline/docs/run-disposables) where it strongly recommends to use the method. 
I assume that the documentation needs updating to reflect the new situation in the code.

**Comment by rwjblue** [comment]
> I assume that the documentation needs updating to reflect the new situation in the code.

Yep! We missed updating that when we removed that requirement.

**Comment by nruth** [comment]
Does the docs site need updating/regenerating somehow with recent changes (as per #978)?

https://ember-lifeline.github.io/ember-lifeline/versions/master/ currently crashes and I'm not sure where else in the project to look.

<img width="637" alt="Screenshot 2022-03-07 at 20 05 25" src="https://user-images.githubusercontent.com/26158/157109740-3041ff88-286c-400e-8017-d122bd2d1954.png">

**Comment by BryanCrotaz** [comment]
https://ember-lifeline.github.io/ember-lifeline/docs

Still incorrectly(?) says to use runDisposables()

--- from website:

Using ember-lifeline's equivalent, in this case runTask, can help ensure that any active async is cleaned up once the object is destroyed.

```
import Component from '@ember/component';
import { runTask, runDisposables } from 'ember-lifeline';

export default Component.extend({
  init() {
    this._super(...arguments);

    runTask(
      this,
      () => {
        this.set('date', new Date());
      },
      500
    );
  },

  willDestroy() {
    this._super(...arguments);

    runDisposables(this);
  },
});
```

**Comment by scalvert** [comment]
Yep, I haven't updated the docs site yet. I've been a bit busy. I'll try to get to it soon.

**Comment by BryanCrotaz** [comment]
Still appears to be incorrect in the readme
